.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - run: mkdir -p docs/_build/html
+      - run: wget http://sunpy.org/index.html -O docs/_build/html/index.html
+
+      - store_artifacts:
+          path: docs/_build/html
+
+      - run:
+          name: "Built documentation is available at:"
+          command: DOCS_URL="${CIRCLE_BUILD_URL}/artifacts/${CIRCLE_NODE_INDEX}/${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}/docs/_build/html/index.html"; echo $DOCS_URL
+
+
+notify:
+  webhooks:
+    - url: https://giles.cadair.com/circleci
```

Comment:
also change this

Reply:
I agree that this URL has to be changed but I don't know what to. Is this something that needs to be set up for the bot?

Reply:
it has to be the URL of the circleci blueprint. The blueprint listens for circleci webhooks which are sent to the address given in the config. (this URL is my running deployment of Giles)